product_packaging_level/models/product_packaging_level.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2019-2020 Camptocamp (<https://www.camptocamp.com>).
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductPackagingLevel(models.Model):
+    _name = "product.packaging.level"
+    _description = "Level management for product.packaging"
+    _order = "sequence, code"
+
+    def _default_language(self):
+        lang_code = self.env["ir.default"]._get("res.partner", "lang")
+        def_lang_id = self.env["res.lang"]._get_data(code=lang_code).id
+        return def_lang_id or self._active_languages()[0]
```

Comment:
`_active_languages` ​​does not exist in `product.packaging.level`.

Reply:
thanks, it's fixed